asset/css/markbind.css

```diff
@@ -302,3 +302,50 @@ panel[minimized]:not([expanded])::before {
 li.footnote-item:target {
     background-color: #eee;
 }
+
+/* "Copy" code block button */
+pre {
+    position: relative;
+}
+
+pre > .copy-btn-icon {
+    background-color: #b4b4b9;
+    border-radius: 0.25rem;
+    color: #f8f8ff;
+    display: inline-block;
+    font-size: 75%;
+    line-height: 1;
+    padding: 0.25em 0.4em;
+    position: absolute;
+    right: 0.3em;
+    text-align: center;
+    top: 0.4em;
+    vertical-align: baseline;
+    white-space: nowrap;
+}
+
+pre .copy-btn-icon:hover {
```

Comment:
Is there a reason why the above has a has the child selector `>` but not this?

Reply:
Resolved. No reason - forgot to remove the `>` after debugging.